pydis_site/apps/api/models/bot/reminder.py

```diff
@@ -15,6 +15,12 @@ class Reminder(ModelReprMixin, models.Model):
             "If not, it has been sent out to the user."
         )
     )
+    jump_url = models.CharField(
```

Comment:
Is there a specific reason for not using an [URLField](https://docs.djangoproject.com/en/2.2/ref/models/fields/#urlfield) here? Using an `URLField` will mean that URLs in requests will automatically validated by the [URLValidator](https://docs.djangoproject.com/en/2.2/ref/validators/#django.core.validators.URLValidator). Seems like a useful addition to make sure the data we receive is valid.